aws-terraform/README.md

```diff
@@ -38,15 +38,16 @@ For most installations, one `t3.micro` size instance will be adequate to serve I
 
 Below is the overall code structure of the Terraform deployment.
 
-- `deploy` - directory contains deployments/environments and separates them from each other. Example: `development`, `staging`, `testing` and so on.
-    - `variables.tf` - sets global variables for the specific deployment and provider configuration. This file already has default values for some of the variables, but it is recommended to review all of them.
-    - `providers.tf` - AWS and Terraform provider configuration.
-    - `main.tf` - invokes required modules and sets module specific variables.
-    - `output.tf` - prints out some of the resources and values.
-    - `deploy/<new-environment>/user_data/03-default-users.yml` - user configuration, ssh username and key.
-    - `new-environment` - can be created by copying an existing one to a new directory and adjusting `variables.tf`, `main.tf`, `providers.tf` as required.
-    - `module_scim_app` - deploys the following AWS resources and their dependencies: Application Load Balancer (ALB), Auto Scaling Groups (ASG), Identity and Access Management (IAM), instance and load balancer security groups, and a public DNS record. ASG monitors instances and automatically adjusts capacity to maintain steady, predictable performance. Capacity is configured in `main.tf` and instance specific configuration is in `variables.tf`.
-    - `module_scim_infra` - deploys network infrastructure: Virtual Private Cloud, networking (route tables, etc), Route 53, Internet Gateways, NAT Gateways, and their dependencies.
+- `/module_scim_app/` - deploys the following AWS resources and their dependencies: Application Load Balancer (ALB), Auto Scaling Groups (ASG), Identity and Access Management (IAM), instance and load balancer security groups, and a public DNS record.
+- `/module_scim_infra/` - deploys network infrastructure: Virtual Private Cloud, networking (route tables, etc), Route 53, Internet Gateways, NAT Gateways, and their dependencies.
+- `/deploy/{environment}` - contains the variable files which you will need to copy and modify to suit your needs.
+
+Specific files you should review and edit include:
+
+- `variables.tf` - sets global variables for the specific deployment and provider configuration. This file already has default values for some of the variables, but it is recommended to review all of them.
+- `providers.tf` - AWS and Terraform provider configuration.
+- `main.tf` - invokes required modules and sets module specific variables.
+- `output.tf` - prints out some of the resources and values.
 
```

Comment:
Eyeing the diff between the old version and your new version, I think you have nicely clarified what needs to be edited, but have also lost the nice description of the folder structure. 

I like the short heirarchical descriptions in the old version, and your new section of exactly what to edit.

Perhaps a combo of the two? Thoughts?

Reply:
Good idea. I went ahead and made the structure example more similar to the old style, while retaining readability.